### PR TITLE
Feature/upgrade 3 15

### PR DESCRIPTION
**Pull request by MelSumner**
first upgrade to 3.15. more to come.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ has changed. This is very bad and screen reader users hear nothing but silence.
 The goal for this addon is to fill the gaps in embers router. Thank you to
 @patrickfox [for the idea!](https://vimeo.com/117614181)
 
+Compatibility
+------------------------------------------------------------------------------
+
+* Ember.js v3.8 or above
+* Ember CLI v2.13 or above
+* Node.js v8 or above
```
Comment (Robdel12) [inline README.md]: Just curious (and totally nit picky!) should this be bumped from v8 since support was dropped Jan 1st, 2020? https://github.com/nodejs/Release

File: tests/dummy/config/environment.js [hunk 2]
```diff
@@ -9,7 +9,15 @@ module.exports = function(environment) {
...
         // Here you can enable experimental features on an ember canary build
+<<<<<<< HEAD
         // e.g. 'with-controller': true
+=======
+        // e.g. EMBER_NATIVE_DECORATOR_SUPPORT: true
+      },
+      EXTEND_PROTOTYPES: {
+        // Prevent Ember Data from overriding Date.parse.
+        Date: false
+>>>>>>> 5f19f80... v3.0.2...v3.15.1
```
Comment (Robdel12) [inline tests/dummy/config/environment.js]: Looks like there's a conflict 👀

Comment (MelSumner) [reply tests/dummy/config/environment.js]: oops! I'll fix it.

File: README.md (hunk 1 above)
Comment (MelSumner) [reply README.md]: @rwjblue can probably tell us the reason it's still in there, but I forget exactly what the reason is.

Comment (rwjblue) [reply README.md]: Ya, its because our official policy is that we support "all current Node LTS versions on the master branch at all times". In practice this means that ember-cli@3.15 (and 3.16) will continue to list Node 8 as supported.

tldr; if this PR will be considered part of a new major bump, I would drop Node 8 here before doing that

File: .travis.yml [hunk 3]
```diff
@@ -18,31 +18,52 @@ env:
...
     - env: EMBER_TRY_SCENARIO=ember-canary
 
+<<<<<<< HEAD
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
   - yarn install --no-lockfile --non-interactive
+=======
```
Comment (Robdel12) [inline .travis.yml]: Conflict here 👀

Comment (MelSumner) [reply .travis.yml]: hmm I wonder why my build isn't catching the conflicts. I will try to fix this root cause and also double check the rest of the files for issues.

**Comment by KamiKillertO** [comment]
#14 has been merged, this PR can be closed now.
